db/seeds.rb

```diff
@@ -3,54 +3,81 @@
 #
 # Examples:
 #
-
-# Begin:Two users that would create a match
-
-    u1 = User.create(username: "fakeuser1" , email: "fakeusersemail1@user.com", password: "fakepass", gender: "male", about_me: "I may be fake but I still wish i could learn skills for free!")
-    u2 = User.create(username: "fakeuser2" , email: "fakeusersemail2@user.com", password: "fakepass", gender: "male", about_me: "I may be fake but I still wish i could learn skills for free!")
-    skill1 = Skill.create(name: "Japanese", category: "Language" , description: "A language spoken in Japan" , url: "http://en.wikipedia.org/wiki/Japanese_language")
-    skill2 = Skill.create(name: "Ruby", category: "Programming Languages" , description: "A elegant and syntatically clear programming language." , url: "http://en.wikipedia.org/wiki/Japanese_language")
-    us1 = HadSkill.create(skill_id: skill1.id, user_id: u1.id, expertise_level: "Novice")
-    us2 = HadSkill.create(skill_id: skill1.id, user_id: u1.id)
-    us3 = HadSkill.create(skill_id: skill2.id, user_id: u2.id, expertise_level: "Novice")
-    us4 = HadSkill.create(skill_id: skill2.id, user_id: u2.id)
+    e_pass = "$2a$10$cHyiV6SD/vdO674gzqZCP.mbSMjcct.gU..aS2Vi7nA.DZ8bILjZy"
+    u1 = User.find_or_create_by(username: "fakeuser1",
+                                email: "fakeusersemail1@user.com",
+                                encrypted_password: e_pass,
+                                gender: "male",
+                                about_me: "I may be fake but I still wish" +
+                                          "i could learn skills for free!")
+    u2 = User.find_or_create_by(username: "fakeuser2",
+                                email: "fakeusersemail2@user.com",
+                                encrypted_password: e_pass,
+                                gender: "male",
+                                about_me: "I may be fake but I still wish i could" +
+                                          "learn skills for free!")
+    skill1 = Skill.find_or_create_by(name: "Japanese",
+                                    category: "Language",
+                                    description: "A language spoken in Japan",
+                                    url: "http://en.wikipedia.org/wiki/Japanese_language")
+    skill2 = Skill.find_or_create_by(name: "Ruby",
+                                    category: "Programming Languages" ,
+                                    description: "A elegant and syntatically clear programming language.",
```

Comment:
Line is too long. [106/80]<br>Align the elements of a hash literal if they span more than one line.